src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/GrpcJsonTranscodingServiceExtensions.cs

```diff
@@ -45,6 +48,14 @@ public static IGrpcServerBuilder AddJsonTranscoding(this IGrpcServerBuilder buil
         }
 
         builder.Services.Configure(configureOptions);
+        builder.Services.AddTransient<IConfigureOptions<GrpcJsonTranscodingOptions>>(services =>
```

Comment:
Should this be `TryAddEnumerable`?
Examples:
https://source.dot.net/#Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs,150
https://source.dot.net/#Microsoft.AspNetCore.SignalR/SignalRDependencyInjectionExtensions.cs,51

Reply:
It seems to work ok as is, and registering this callback twice won't have a negative effect.

But I'll defer to prior art and change it 😄